includes/rest/class-followers.php

```diff
@@ -115,9 +136,33 @@ public static function request_parameters() {
 			'default' => 1,
 		);
 
+		$params['per_page'] = array(
+			'type' => 'integer',
+			'default' => 10,
+		);
+
+		$params['order'] = array(
+			'type'    => 'string',
+			'default' => 'desc',
+			'enum'    => array( 'asc', 'desc' ),
+		);
+
 		$params['user_id'] = array(
 			'required' => true,
 			'type' => 'string',
+			'validate_callback' => function( $param, $request, $key ) {
```

Comment:
this will be implicitly checked with

```PHP
$user    = User_Collection::get_by_various( $user_id );

if ( is_wp_error( $user ) ) {
	return $user;
}
```

I would remove the check, because otherwise we might ran into inconsistencies.